pkgs/by-name/ob/obj2voxel/package.nix

```diff
@@ -67,6 +67,7 @@ stdenv.mkDerivation (finalAttrs: {
   meta = with lib; {
     description = "Convert OBJ and STL files to voxels, with support for textures";
     mainProgram = "obj2voxel";
+    homepage = "https://github.com/Eisenwave/obj2voxel/blob/master/README.adoc";
```

Comment:
```suggestion
    homepage = "https://github.com/Eisenwave/obj2voxel";
```

Reply:
Any rationale for this requested change? When looking for a _homepage_, people usually expect a page where they can read about the project, and not a list of files (which is what is presented when just opening the repo URL). So the README file is what comes closest to a homepage.

Reply:
The README is right there when scrolling down, so the request actually allows both users and developers get what they want.

Reply:
Changed it to point right to the repo as this is how many other packages also do it currently. I still think repo and homepage are not synonymous, and there ought to be an extra field, with the current meaning of "homepage" with that fallback being emulated by doing a `meta.homepage or meta.repo`.

Reply:
I sometimes abuse `meta.downloadPage` for the repo url.